src/content-service/models/product.ts

```diff
@@ -44,6 +47,10 @@ export class Product {
   getVenue () {
     return this.venue;
   }
+  
+  getPosterImageImageUrl () {
```

Comment:
Is this intended to have two `Image`'s? maybe an accidental dupe? :)

Reply:
copy pasta issues 😁 fixed!

Reply:
haha messy pasta 🍝